### DISCUSSION TITLE
Pick window

### DISCUSSION THREAD

**Discussion by chgara** [nvim-tree/nvim-tree.lua#1840] [Q&A] [answered]
### Description

When splitting window into half or more and I select a new file to open it pop ups: "Pick window:" , I saw in the that can not be automated how the window is picked.

So if I have two windows | A | B | and I am in B and open nvim-tree I want to open it on B automatically like nerd-tree does. But config does not allow it.

### Neovim version

```text
NVIM v0.8.1
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

Manjaro latest stable

### nvim-tree version

29788cc

### Minimal config

```lua
No config need, default one
```

### Steps to reproduce

1. Open file
2. Split and open another, so now you have tab |  A  |  B  |
3. Open nvim-tree and select a file, it will ask

**Comment by alex-courtis** [comment]
This is not a bug. Please seek first to understand.

Disabling `:help nvim-tree.actions.open_file.window_picker.enable` should result in you desired behaviour.

See `:help nvim-tree.actions.open_file.window_picker.picker`. This allows you to customise your window selection. Here's an example of a custom function: https://github.com/nvim-tree/nvim-tree.lua/pull/1782#issuecomment-1330039322

Moving to discussion.

**Reply by redx177** [reply, marked as answer]
Thank you for clarifying. I was confused as well because I missed the window identifiers in the window's status line. 
The help command doesn't work anymore. It is `:help nvim_tree.config.actions.open_file.window_picker` now
